src/main/java/seedu/address/logic/commands/DeleteProcCommand.java

```diff
@@ -33,7 +33,7 @@
             + "INDEX (must be a positive integer and an existing Procedure)\n"
             + "Example: " + COMMAND_WORD + " 1 2";
 
-    public static final String MESSAGE_EDIT_CLIENT_SUCCESS = "Current Procedure List: %1$s";
+    public static final String MESSAGE_DELETE_PROCEDURE_SUCCESS = "Deleted Procedure: %1$s";
```

Comment:
I really like your implementation of alerting to the user which Procedure has been deleted!